wikidata_game/stats.php

```diff
@@ -1,21 +1,28 @@
 <?php declare(strict_types=1);
 
+
+use ReferenceIsland\MatchesRepository;
+
+require __DIR__ . '/vendor/autoload.php';
+
 require_once('includes/constants.php');
 require_once('includes/responses.php');
 require_once('includes/setup-db.php');
 
+$matchesRepository = new MatchesRepository($db);
```

Comment:
To maintain the pattern I think this should actually go in `setup-db.php` (And probably the also the autoloader above)

Reply:
I *think* the autoloader should go in the entrypoint (which in this case is stats.php) and the db-setup should probably be also autoloaded (probably as a class with a static method) and not specifically reference by file path.